docs/design/proposals/encryption-with-keyprotect.md

```diff
@@ -56,7 +56,7 @@ config map to `KMS_SERVICE_NAME`.
 
 ## Volume Encrypt or Decrypt Operation
 
-The IBM Key protect server's `wrap` and `unwrap` functionalities will be used by
+The IBM Key Protect server's `wrap` and `unwrap` functionalities will be used by
```

Comment:
not only protect, but Key also need to be capitalized as per https://cloud.ibm.com/docs/key-protect?

Reply:
yep, done